From linux-kernel-owner+greg=40kroah.com-S2992513AbXCWQPL@vger.kernel.org  Mon Mar 26 14:00:57 2007
Date: Fri, 23 Mar 2007 12:15:05 -0400
From: Prarit Bhargava <prarit@redhat.com>
To: linux-kernel@vger.kernel.org, <akpm@linux-foundation.org>,
	<tom.l.nguyen@intel.com>, <kristen.c.accardi@intel.com>,
	<ddutile@redhat.com>
Cc: Prarit Bhargava <prarit@redhat.com>
Message-Id: <20070323161505.10285.9944.sendpatchset@prarit.boston.redhat.com>
Subject: PCI: Fix warning message in PCIE port driver

PCIE error output should conform to vendor_id:device_id.

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/pcie/portdrv_pci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/pci/pcie/portdrv_pci.c
+++ b/drivers/pci/pcie/portdrv_pci.c
@@ -93,7 +93,7 @@ static int __devinit pcie_portdrv_probe 
         if (!dev->irq && dev->pin) {
 		printk(KERN_WARNING 
 		"%s->Dev[%04x:%04x] has invalid IRQ. Check vendor BIOS\n", 
-		__FUNCTION__, dev->device, dev->vendor);
+		__FUNCTION__, dev->vendor, dev->device);
 	}
 	if (pcie_port_device_register(dev)) {
 		pci_disable_device(dev);
